### PR TITLE
feat(badger-api): Add graphql-codegen and generate protocol schemas

### PR DESCRIPTION
**Pull request by ethkaneki**
* Add `dotenv` and create `.env.development` file to store some environment variables. 
* Add `graphql-codegen` and pull in schemas from Badger, MasterChef, SushiSwap, and UniSwap subgraphs

### CODE REVIEW

File: src/graphql/generated/badger.ts [hunk 1]
```diff
@@ -0,0 +1,763 @@
...
+
+export type UserSettBalancesArgs = {
+	skip?: Maybe<Scalars['Int']>;
+	first?: Maybe<Scalars['Int']>;
+	orderBy?: Maybe<UserSettBalance_OrderBy>;
+	orderDirection?: Maybe<OrderDirection>;
+	where?: Maybe<UserSettBalance_Filter>;
+};
+
+export type UserSettBalance = {
```
Comment (axejintao) [inline src/graphql/generated/badger.ts:592]: okay, this is awesome.

**Review by axejintao — COMMENTED** [review]
so the subgraphs are (probably) the same - but can we swap out the croco-finance one on the .env and serverless with 
`https://thegraph.com/explorer/subgraph/jiro-ono/sushiswap-exchange`
and then we will also need to add the currently deployed badger subgraph which is 
`https://thegraph.com/explorer/subgraph/m4azey/badger-test`
the one we have in there is mine so kind of weird, but the `badger-test` one has the harvest data we will need for the revenue stuff.

File: src/graphql/generated/badger.ts (hunk 1 above)
Comment (ethkaneki) [reply src/graphql/generated/badger.ts]: Fixed

File: serverless.yml [hunk 2]
```diff
@@ -51,9 +51,9 @@ provider:
     shouldStartNameWithService: true
     minimumCompressionSize: 1024
   environment:
-    BADGER: "https://api.thegraph.com/subgraphs/name/axejintao/badger-dao"
+    BADGER: "https://api.thegraph.com/subgraphs/name/m4azey/badger-test"
```
Comment (axejintao) [inline serverless.yml:54]: sorry, i think maybe i wasn't clear - these are both needed

**Pull request merged**